components/core/src/metrics.rs

```diff
@@ -0,0 +1,221 @@
+// Copyright (c) 2016 Chef Software Inc. and/or applicable contributors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use std::thread;
+use std::sync::mpsc::{channel, sync_channel, Sender, Receiver, SyncSender};
+use std::sync::{Once, ONCE_INIT};
+use statsd::Client;
+use env;
```

Comment:
Could you lexically sort and arrange the imports here? I'm struggling to find the style docs we've been unofficially woking off, but generally speaking we have 3 stanzas/group of imports: 1) `std::*` imports, 2) extern crate imports, 3) current crate imports ([sup example](https://github.com/habitat-sh/habitat/blob/23afffd3ef53a80def00731702faa53faf635d9a/components/sup/src/manager/service_updater.rs#L15-L35), [rust/fmt-rfcs example](https://github.com/rust-lang-nursery/fmt-rfcs/blob/b4c703ce19441089e065775a26a37dbf4150aad5/example/lists.rs#L11-L18))